core/src/main/java/io/questdb/cairo/TableReader.java

```diff
@@ -841,6 +850,11 @@ boolean isColumnCached(int columnIndex) {
         return symbolMapReaders.getQuick(columnIndex).isCached();
     }
 
+    private boolean isMetaFileMissingFileSystemError(CairoException ex) {
+        int errno = ex.getErrno();
+        return errno == CairoException.ERRNO_FILE_DOES_NOT_EXIST || errno == CairoException.METADATA_VALIDATION;
```

Comment:
windows errno might be different

Reply:
Missing file error is still 2, tested on windows in unit test `testMetadataFileDoesNotExist`